prysm.ps1

```diff
@@ -3,7 +3,6 @@ $ProgressPreference = 'SilentlyContinue' # Disable Invoke-WebRequest progress ba
 
 # Complain if invalid arguments were provided.
 if ("beacon-chain", "validator", "slasher" -notcontains $args[0]) {
-    Write-Host "ERROR: prysm is only supported on 64-bit Operating Systems" -ForegroundColor Red;
```

Comment:
Any reason to remove this?

Reply:
The error should only show up if you don't have a 64 bit system, which is what is being checked below this piece of code at line 30. Showing this "error" in full red confuses the user if he's already on a 64 bit system, which makes him or her think there's something wrong when there isn't and this message can safely be ignored.